extract-jvm

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+if [ -z $JAVA_HOME ]; then
+    echo -e "Please define JAVA_HOME and try again"
+    exit 1
+fi
+
+mkdir -p build/jvm
+cp $JAVA_HOME/lib/modules build/jvm
+
+cd build/jvm
+jimage extract modules
```

Comment:
i also don't have _any_ `.jimage` files on my system, which might be from having a post-modules jdk. instead all the good stuff looks to be in `/usr/lib/jvm/java-11-openjdk-amd64/jmods`, where `java.base.jmod` has String and friends.

if this is in fact from having an older jdk on your system, i'm good with this, and i'll tweak this script later to play nice with modules as well.

Reply:
They aren't postfixed with .jimage; run `file /usr/lib/jvm/java-11-openjdk-amd64/lib/modules`

Reply:
oh yep, i'd expected `modules` to be a directory and totally glanced over it. cool.